response/slack/incident_notifications.py

```diff
@@ -28,16 +32,36 @@ def remind_incident_lead(incident: Incident):
     except CommsChannel.DoesNotExist:
         pass
 
+@recurring_notification(interval_mins=30, max_notifications=10)
+def remind_update(incident: Incident):
+    update = StatusUpdate.objects.filter(incident=incident).order_by("timestamp").first()
+    if update is not None:
+        if update.timestamp < datetime.datetime.now()-datetime.timedelta(minutes=60):
+            logger.info("The last update was over 1h ago")
+            try:
+                comms_channel = CommsChannel.objects.get(incident=incident)
+                if not incident.is_closed():
+                    user_to_notify = incident.lead
+                    settings.SLACK_CLIENT.send_ephemeral_message(
+                        comms_channel.channel_id,
+                        user_to_notify.external_id,
+                        "The last update was over 1h ago. To provide a new update use the command:\n `/%s update [text]`"% settings.SLACK_SLASH_COMMAND,
+            )
+            except CommsChannel.DoesNotExist:
+                pass
```

Comment:
Can't we log (import logging :-) something here instead of pass?

Maybe there is already something logged, but I'm asking just to check :-)